IPs/RSKIP432.md

```diff
@@ -0,0 +1,69 @@
+---
+rskip: 432
+title: RbtcSwap
```

Comment:
Can we have a little bit more descriptive name? Something like "Add new RBTCSwap contract"

Reply:
How about the name `RbtcSwap mechanism`? This feature is not a new contract, but an implementation of a mechanisim that uses an existing contact

Reply:
But it's a hardcoded bytecode contract that you invoke how ?

Reply:
Sorry if it got confusing, what I meant is that this RSKIP is not only to add the contract but also the mechanism that allows the claim to be paid after its execution.